core/templates/dev/head/css/oppia.css

```diff
@@ -162,42 +165,12 @@ body {
   margin-top: 16px;
 }
 
-.oppia-profile-tabs {
+.oppia-profile-content-card {
   background-color: #fff;
-  margin: 0;
-  padding: 0;
-}
-
-.oppia-profile-tabs-content {
-  padding: 24px 24px 48px 24px;
-}
-
-.oppia-profile-tabs md-content h1:first-child {
-  margin-top: 0;
-}
-
-.oppia-profile-tabs md-tab {
-  background-color: #009688;
-  border: 1px solid rgba(0,0,0,0.1);
-}
-
-.oppia-profile-tabs md-tabs.md-default-theme md-tab:focus {
-  border: 1px solid rgba(0,0,0,0.1);
-}
-
-.oppia-profile-tabs md-tabs-ink-bar {
-  background: #0D47A1 !important;
-  /* Without this, the ink bar does not show up on initial load. */
-  /* TODO(sll): Find a better fix for this. The current fix has an
-    ink bar that is too large on small-width screens.
-  */
-  display: block !important;
-  height: 6px;
-  width: 266px;
-}
-
-.oppia-profile-tabs md-tab-label {
-  color: #ffffff;
+  width: 520px;
```

Comment:
You might want to make this slightly smaller -- there is an anomaly when the window is at width 800px and the height is small. (The scrollbar forces the right card to the bottom.)